tests/iDibo_tests.py

```diff
@@ -1,11 +1,35 @@
-from nose.tools import *
-import iDibo
+import unittest
+from gluon.shell import exec_environment
+from gluon.contrib.test_helpers import form_postvars
```

Comment:
I'll check to confirm I committed the file. I'll appreciate u look into the last question I asked on the main project about the db and contoller for listing items in campaigns table. Thanks



Best Regards,
'Yemisi

Sent from my Samsung TabletAna Oprea <notifications@github.com> wrote:In tests/iDibo_tests.py:

> @@ -1,11 +1,35 @@
> -from nose.tools import *
> -import iDibo
> +import unittest
> +from gluon.shell import exec_environment
> +from gluon.contrib.test_helpers import form_postvars
Where can I find this module? I don't see it in https://github.com/web2py/web2py/tree/master/gluon/contrib

Do you also get the error below?
$ python web2py.py -S iDibo -M -R applications/iDibo/tests/iDibo_tests.py 
Version 2.9.11-stable+timestamp.2014.09.15.23.35.11
Traceback (most recent call last):
File "/home/ana/dev/learnitgirl/web2py/gluon/shell.py", line 262, in run
execfile(startfile, _env)
File "applications/iDibo/tests/iDibo_tests.py", line 3, in <module>
from gluon.contrib.test_helpers import form_postvars
File "/home/ana/dev/learnitgirl/web2py/gluon/custom_import.py", line 105, in custom_importer
return NATIVE_IMPORTER(name, globals, locals, fromlist, level)
ImportError: No module named test_helpers

—
Reply to this email directly or view it on GitHub.